felix/fv/bpf_test.go

```diff
@@ -945,6 +1046,30 @@ func describeBPFTests(opts ...bpfTestOpt) bool {
 
 					pol = createPolicy(pol)
 
+					By("allowing to self via MASQ", func() {
+
+						nets := []string{felixes[0].IP + "/32"}
+						switch testOpts.tunnel {
+						case "ipip":
+							nets = []string{felixes[0].ExpectedIPIPTunnelAddr + "/32"}
+						}
+
+						pol := api.NewGlobalNetworkPolicy()
+						pol.Namespace = "fv"
```

Comment:
GNPs aren't namespaced; surprised it doesn't reject it!

Reply:
In iptables mode, I think traffic to self via a service does require a policy but the policy sees the Pod IP.  I don't think it's acceptable to require a policy that matches the post-SNAT IP since that's different (and the IP is hard to guess)

Reply:
OK, this is not an issue created by this PR, so I'd prefer to address it separately. WDYT

Reply:
Please drop a FIXME explaining that this policy shouldn't be needed.